website/api_versioned_docs/version-0.21/zkvm/precompiles.md

```diff
@@ -42,6 +42,16 @@ can be in the [ECDSA example][ecdsa]. Note the [use of the patched
 versions][ecdsa-patched] of `sha2`, `crypto-bigint` and `k256` crates used in
 the guest's `Cargo.toml`.
 
+Some crates will allow you a choice of different backend crates, and you may be
```

Comment:
hmm i think you meant to make this change in `api_versioned_docs/version-1.2`

Reply:
Whoops! Good call